packages/stryker-javascript-mutator/src/helpers/BabelParser.ts

```diff
@@ -28,14 +29,15 @@ export default class BabelParser {
     return ast;
   }
 
-  static getNodes(ast: babel.types.File): babel.types.Node[] {
-    const nodes: babel.types.Node[] = [];
+  static getNodes(ast: babel.types.File): NodeWithParent[] {
+    const nodes: NodeWithParent[] = [];
 
     babel.traverse(ast, {
       enter(path: NodePath<babel.types.Node>) {
         const node = path.node;
```

Comment:
If you do the cast here, it is only needed once

Reply:
I was having issues with this cast because the value that I assign does not have a parent property, that's on `path.parent`

Reply:
Good point. But at the same time, the first node will not have a parent (undefined). So it might need to be optional, right?

Reply:
Good point! I'll make it optional

Reply:
Good point! You do that.

(couldn't help myself)